tests/tracer/test_ext.py

```diff
@@ -21,7 +25,154 @@ def _updateenv(monkeypatch, env):
         monkeypatch.setenv(str(k), str(v))
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    cwd = str(tmpdir)
+    subprocess.check_output("git init", cwd=cwd, shell=True)
+    subprocess.check_output('git remote add origin "git@github.com:test-repo-url.git"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.name "John Doe"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.email "john@doe.com"', cwd=cwd, shell=True)
+    subprocess.check_output("touch tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output("git add tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output(
+        'git commit --date="2021-01-19T09:24:53-0400" -m "this is a commit msg" --no-edit', cwd=cwd, shell=True
+    )
+    subprocess.check_output(
+        'GIT_COMMITTER_DATE="2021-01-20T04:37:21-0400" git -c user.name="Jane Doe" -c user.email="jane@doe.com" commit '
+        "--amend --no-edit",
+        cwd=cwd,
+        shell=True,
+    )
+    yield cwd
+
+
+@pytest.fixture
+def git_repo_error(tmpdir):
```

Comment:
docstring/comment, it isn't clear on why this is an error

Reply:
Good catch, I'll add docstrings for this 😄